Sources/Basics/Observability.swift

```diff
@@ -236,6 +261,14 @@ extension DiagnosticsEmitterProtocol {
             return false
         }
     }
+
+    private func makeMessage(from message: String, underlyingError: Error?) -> String {
```

Comment:
nit: doc comment
```suggestion
    /// If `underlyingError` is not `nil`, its human-readable description is interpolated with `message`,
    /// otherwise `message` itself is returned.
    private func makeMessage(from message: String, underlyingError: Error?) -> String {
```